packages/php-storage-driver-common/proto/table.proto

```diff
@@ -460,6 +460,25 @@ message TableExportToFileResponse {
  * Command is used to remove specific rows from table based on supplied constraints
  */
 message DeleteTableRowsCommand {
+  message WhereRefTableFilter {
+    enum Operator {
+      IN = 0;
+      NOT_IN = 1;
+    }
+
+    string columnName = 1; // column where is filter applied
+
+    ImportExportShared.DataType columnDataType = 2; // datatype of value usually driver will cast value with TRY method
+
+    Operator operator = 3;
+
+    string refPath = 4;
```

Comment:
tohle ma byt pole. protoze pro BQ to bude [ dataset ], zatimco pro SNFLK to muze byt [ database, schema ]